src/codeu/chat/common/LogoutCallback.java

```diff
@@ -0,0 +1,4 @@
+package codeu.chat.common;
+public interface LogoutCallback{
```

Comment:
I get the idea of these Callback classes but I don't think we need such a simple interface, especially when it has one single method and is implemented only once. Maybe we could simply add the method to the class that implements it and get rid of this interface. That way we don't end up with a lot of unnecessary classes.